guideforms.py

```diff
@@ -335,6 +335,20 @@
         ('If your feature was available as an Origin Trial, link to a summary '
          'of usage and developer feedback. If not, leave this empty.')),
 
+    'i2e_lgtms': forms.EmailField(
+        required=False, label='i2e LGTM by',
+        widget=forms.EmailInput(
+            attrs={'multiple': True, 'placeholder': 'email'}),
+        help_text=('Full email address of API owner who LGTM\'d the '
```

Comment:
Who's supposed to enter them? The feature owner or the API owner who gave the LGTM?

Reply:
The feature owners for now.  Eventually we will have API owners click an LGTM button, but that will come later.

Reply:
My strong suspicion is that feature owners will not come back to the tool to do this.  It's manual work (copying and pasting from the email thread) that won't help them move the feature along (i.e., there's no enticement).

If we pick this path, over a checkbox, an LGTM button (even in a light weight form) will likely have to come pretty quickly.